addons/mrp/mrp.py

```diff
@@ -151,6 +151,10 @@ class mrp_bom(osv.osv):
     _description = 'Bill of Material'
     _inherit = ['mail.thread']
 
+    def _get_bom_ids(self, cr, uid, product_template_ids, context=None):
+        # It will return BOM ids of product.
+        return map(int, self.pool['product.template'].browse(cr, uid, product_template_ids, context=context).bom_ids)
```

Comment:
@gan-odoo 
this is function to use to get bom_ids of particular but we should use direct search method of mrp.bom instead of browse.